packages/app/src/systems/Account/components/BalanceWidget/BalanceWidget.tsx

```diff
@@ -12,6 +12,8 @@ import type { Account } from '@fuel-wallet/types';
 import { useEffect, useState } from 'react';
 import type { ReactNode } from 'react';
 
+import type { AccountInputs } from '../..';
```

Comment:
inside the components, you should import from the components folder to avoid cycle dependencies.